src/tabs/presets/presets.js

```diff
@@ -359,13 +359,27 @@ TABS.presets.updateSearchResults = function() {
             searchString: this._inputTextFilter.val().trim(),
         };
 
+        this.updateSelectStyle();
         searchParams.authors = searchParams.authors.map(str => str.toLowerCase());
-
         const fitPresets = this.getFitPresets(searchParams);
         this.displayPresets(fitPresets);
     }
 };
 
+TABS.presets.updateSelectStyle = function() {
+    this.updateSingleSelectStyle(this._selectCategory);
+    this.updateSingleSelectStyle(this._selectKeyword);
+    this.updateSingleSelectStyle(this._selectAuthor);
+    this.updateSingleSelectStyle(this._selectFirmwareVersion);
+    this.updateSingleSelectStyle(this._selectStatus);
+};
+
+TABS.presets.updateSingleSelectStyle = function(select) {
+    const selectedOptions = select.multipleSelect("getSelects", "text");
+    const isSomethingSelected = (0 !== selectedOptions.length);
```

Comment:
Just cast this to boolean instead of yoda comparisons 😉 
```suggestion
    const isSomethingSelected = !!selectedOptions.length;
```